### ISSUE TITLE
Add mechanism to add scheduling constraints when running emit_signal (Trac #68)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#72]
This ticket takes over the second issue in ticket #25.

The idea is to provide a way to ensure plugins are processed in the correct order upon an emit_signal() call.  

Migrated from http://microfluidics.utoronto.ca/ticket/68
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "This ticket takes over the second issue in ticket #21.\n\nThe idea is to provide a way to ensure plugins are processed in the correct order upon an emit_signal() call.  ", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "Add mechanism to add scheduling constraints when running emit_signal", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-03-14T15:41:14", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120314T15:43:47: 
* christian@fobel.net commented:

> commit 216bbcea9d630ec6ec085c718e766e896704cad6
> Merge: 86478003b0c4cf5f52fd65e3c5158d9513166902 b3973d8c7fe827bf1991a06f89a66f318ebb6242
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Mar 14 11:19:23 2012 -0400
> 
>     [SUBMODULE] Merge branch 'scheduled_plugins'...
>     
>     [SUBMODULE] Merge branch 'scheduled_plugins'.
>     
>     Fixes #72.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
